site/src/_pages/docs/3_spec/1_core-specification.mdx

```diff
@@ -1,6 +1,6 @@
-# Core Specification 0.2
+# Core Service
```

Comment:
Hmm actually I’m not sure if the Core is a ‘service’. We should probably keep ‘Core Specification’ or switch to something like ‘Core’ /  ‘BlockProtocol Core’.

In the latter case we should probably update URLs too, e.g.:

- `/docs/spec/core`
- `/docs/spec/graph-service`

I can do that (including new redirects) based on the decision.

Reply:
Agree with not calling core a 'service' – if we want to do that we need to rework the rest of the document, which specifies how you can define 'services', which are described in a way which is inconsistent with this document.

We can do that, just needs a bit more thinking about how we then frame the different parts of the BP – we no longer have the core spec which defines rendering and comms transport, and service spec which defines comms content, but instead a collection of services, one of which has special status.

Reply:
Actioned in https://github.com/blockprotocol/blockprotocol/pull/518/commits/6b363a9d924bff39d2f1666a2b73a38d8073b2a2